src/lib/routes/client-api/feature.ts

```diff
@@ -51,10 +64,40 @@ export default class FeatureController extends Controller {
         this.featureToggleServiceV2 = featureToggleServiceV2;
         this.segmentService = segmentService;
         this.clientSpecService = clientSpecService;
+        this.openApiService = openApiService;
         this.logger = config.getLogger('client-api/feature.js');
 
-        this.get('/', this.getAll);
-        this.get('/:featureName', this.getFeatureToggle);
+        this.route({
+            method: 'get',
+            path: '/:featureName',
+            handler: this.getFeatureToggle,
+            permission: NONE,
+            middleware: [
+                openApiService.validPath({
+                    operationId: 'getFeature',
```

Comment:
Should make sure these operationIds are unique, and that they match the handler name.

Maybe `getClientFeature` could work here?